package.json

```diff
@@ -54,7 +54,6 @@
     "@salesforce/dev-scripts": "^9.0.0",
     "@salesforce/plugin-command-reference": "^3.0.76",
     "eslint-plugin-sf-plugin": "^1.18.0",
-    "jsforce": "^2.0.0-beta.29",
```

Comment:
should include jsforce-node in devdeps

Reply:
done.
there's no lockfile changes because it's already there 👍🏼